site/content/en/guides/consumer/function/export/circleci/_index.md

```diff
@@ -105,11 +105,11 @@ workflows:
 
 If you don’t have one yet, you can just copy the exported `config.yml` file into `.circleci/config.yml` in your project root. Then do the steps above to make the pipeline fully functional. Once all changes are pushed into GitHub, you can do the following steps to setting up your project on CircleCI:
 
-1.  Log into [CircleCI](https://circleci.com/) and choose `Log In with GitHub`.
-1.  Select an organization if prompted.
-1.  Choose your newly created repo and click `Set Up Project`.
-1.  Click `Add Manually` since you have already added `.circleci/config.yml`.
-1.  Click `Start Building`.
+1. Log into [CircleCI](https://circleci.com/) and choose `Log In with GitHub`.
```

Comment:
Is this supposed to be a numbered list?

Reply:
Yes, the numbers get rendered correctly later.